Docs: missing aria-hidden on some decorative SVGs

hannahiss opened the pull request:
### Description

<!-- Describe your changes in detail -->
Add aria-hidden on decorative SVGs to make sure they are ignored by assistive technologies.

### Motivation & Context

<!-- Why is this change required? What problem does it solve? -->

### Type of changes

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

#### Live previews

<!-- Please add direct links where your modifications can be seen in the documentation -->

- <https://deploy-preview-40756--twbs-bootstrap.netlify.app>
- <https://deploy-preview-40756--twbs-bootstrap.netlify.app/docs/components/tooltips/>
- <https://deploy-preview-40756--twbs-bootstrap.netlify.app/docs/examples/features/>
- <https://deploy-preview-40756--twbs-bootstrap.netlify.app/docs/utilities/position/>

### Related issues

<!-- Please link any related issues here. -->

hannahiss committed:
Docs: missing aria-hidden on some decorative SVGs

hannahiss committed:
Docs: missing aria-hidden on check mark on docs version dropdown in docs' header

hannahiss committed:
Docs: missing aria-hidden on check mark on theme toggler dropdown in docs' header

hannahiss committed:
Docs: missing aria-hidden on position utility example

hannahiss committed:
Docs: missing aria-hidden on features examples

hannahiss committed:
Docs: missing aria-hidden

hannahiss committed:
Docs: missing label on link

anandfresh reviewed (commented):
LGTM

patrickhlauke commented on site/content/docs/5.3/examples/features/index.html:
```diff
@@ -66,35 +66,35 @@ <h2 class="pb-2 border-bottom">Columns with icons</h2>
       <div class="feature col">
         <div class="feature-icon d-inline-flex align-items-center justify-content-center text-bg-primary bg-gradient fs-2 mb-3">
-          <svg class="bi" width="1em" height="1em"><use xlink:href="#collection"/></svg>
+          <svg class="bi" width="1em" height="1em" aria-hidden="true"><use xlink:href="#collection"/></svg>
```
note that `aria-hidden="true"` is unnecessary if the svg doesn't contain any text elements. SRs will ignore them the same way they ignore a series of purely empty presentational `<div>`s and `<span>`s

patrickhlauke replied:
(just as a heads-up before you laboriously go and add it to *every*, *single*, *svg*)

hannahiss replied:
Thanks for your answer :-) Our accessibility experts asked us to add aria-hidden on decorative SVGs in our Boosted doc, so I did it on Bootstrap when relevant. There aren't so much in the doc, but a lot more in the examples...! So if you're ok with this I can stop there and submit the PR?

patrickhlauke replied:
wonder what the rationale provided by the accessibility experts there was.

personally, I have no problem with having all the `aria-hidden`s added, it just felt a bit wasteful/pointless. but *if* we're going to add them, then I'd say it should cover all instances, as it would be more confusing if we had a mix of "some `<svg>`s do have it, others don't". so i'd say either remove the ones you did from this PR, or...go through all the rest

patrickhlauke commented on site/content/docs/5.3/components/tooltips.md:
```diff
@@ -111,8 +111,8 @@ And with custom HTML added:
 <div class="bd-example tooltip-demo">
-  <a href="#" class="d-inline-block" data-bs-toggle="tooltip" data-bs-title="Default tooltip">
-    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 100 100">
+  <a href="#" class="d-inline-block" data-bs-toggle="tooltip" data-bs-title="Default tooltip" aria-label="Hover or focus to see default tooltip">
```
that's an embarassing omission in the docs, thanks for catching that

dependabot[bot] committed:
Build(deps-dev): Bump rollup from 4.20.0 to 4.21.1 (#40775)

Bumps [rollup](https://github.com/rollup/rollup) from 4.20.0 to 4.21.1.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.20.0...v4.21.1)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump @babel/preset-env from 7.25.3 to 7.25.4 (#40773)

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.25.3 to 7.25.4.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.4/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump hugo-bin from 0.129.1 to 0.130.1 (#40774)

Bumps [hugo-bin](https://github.com/fenneclab/hugo-bin) from 0.129.1 to 0.130.1.
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.129.1...v0.130.1)

---
updated-dependencies:
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

XhmikosR committed:
npm audit fix

dependabot[bot] committed:
Build(deps-dev): Bump stylelint-config-twbs-bootstrap (#40786)

Bumps [stylelint-config-twbs-bootstrap](https://github.com/twbs/stylelint-config-twbs-bootstrap) from 14.2.0 to 15.0.0.
- [Release notes](https://github.com/twbs/stylelint-config-twbs-bootstrap/releases)
- [Commits](https://github.com/twbs/stylelint-config-twbs-bootstrap/compare/v14.2.0...v15.0.0)

---
updated-dependencies:
- dependency-name: stylelint-config-twbs-bootstrap
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump rollup from 4.21.1 to 4.21.2 (#40785)

Bumps [rollup](https://github.com/rollup/rollup) from 4.21.1 to 4.21.2.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.21.1...v4.21.2)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump eslint-plugin-import from 2.29.1 to 2.30.0 (#40783)

Bumps [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) from 2.29.1 to 2.30.0.
- [Release notes](https://github.com/import-js/eslint-plugin-import/releases)
- [Changelog](https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md)
- [Commits](https://github.com/import-js/eslint-plugin-import/compare/v2.29.1...v2.30.0)

---
updated-dependencies:
- dependency-name: eslint-plugin-import
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump rtlcss from 4.2.0 to 4.3.0 (#40782)

Bumps [rtlcss](https://github.com/MohammadYounes/rtlcss) from 4.2.0 to 4.3.0.
- [Release notes](https://github.com/MohammadYounes/rtlcss/releases)
- [Changelog](https://github.com/MohammadYounes/rtlcss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/MohammadYounes/rtlcss/compare/v4.2.0...v4.3.0)

---
updated-dependencies:
- dependency-name: rtlcss
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump @babel/cli from 7.24.8 to 7.25.6 (#40788)

Bumps [@babel/cli](https://github.com/babel/babel/tree/HEAD/packages/babel-cli) from 7.24.8 to 7.25.6.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.6/packages/babel-cli)

---
updated-dependencies:
- dependency-name: "@babel/cli"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

louismaximepiton committed:
Docs: clarify the title on the 'About > Overview' page (#40762)

dependabot[bot] committed:
Build(deps-dev): Bump postcss from 8.4.41 to 8.4.44 (#40784)

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.41 to 8.4.44.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.41...8.4.44)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: XhmikosR <xhmikosr@gmail.com>

hannahiss committed:
Docs: add missing `$enable-css-grid` in 'Customize > Options' (#40797)

Co-authored-by: Julien Déramond <juderamond@gmail.com>

hannahiss committed:
Remove aria-label from this PR

hannahiss committed:
Rollback unwanted modif

hannahiss commented:
Just for info, after some discussions with the accessibility team, they confirmed that aria-hidden is needed on SVGs until now, even if most of assistive technologies will ignore SVGs that do not contain texts. They rely on criterion [WCAG 1.1.1](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html) saying:

> Non-text Content (Level A)
> Success Criterion (SC)
> All [non-text content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html#dfn-text-alternative) that serves the equivalent purpose, except for the situations listed below.
> [...]
> Decoration, Formatting, Invisible
**> If non-text content is [pure decoration](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html#dfn-pure-decoration), is used only for visual formatting, or is not presented to users, then it is implemented in a way that it can be ignored by [assistive technology](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html#dfn-assistive-technology).**
> 

The last paragraph says we must do what is needed to ensure that decorative elements are ignored, and the only way to do it, is to use `aria-hidden`. 

I'm going to make another PR about the tooltip icon's label (#40808 ) so that we can at least fix this label issue...

dependabot[bot] committed:
Build(deps-dev): Bump postcss from 8.4.44 to 8.4.45 (#40806)

Bumps [postcss](https://github.com/postcss/postcss) from 8.4.44 to 8.4.45.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.44...8.4.45)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

bbonch committed:
Fix popover remaining open when toggled after being shown (#40803)

Co-authored-by: Dmitry Sergienko <dmitry.sergienko@codefirst.net>
Co-authored-by: Julien Déramond <juderamond@gmail.com>

coliff committed:
Docs: update notice on Safari flexbox gap support in 'Helpers > Stacks' (#40844)

hannahiss committed:
Add missing aria-hidden to decorative `<svg>` and `aria-label` and `role="img"` to informative ones in examples until dropdowns

hannahiss committed:
.

hannahiss committed:
Add missing aria-hidden to decorative `<svg>` and `aria-label` and `role="img"` to informative ones in examples until modals

hannahiss committed:
rollback

hannahiss committed:
rollback

coliff committed:
Docs: update `min-height` inline style for modal with long content (#40870)

dependabot[bot] committed:
Build(deps): Bump body-parser in the npm_and_yarn group (#40862)

Bumps the npm_and_yarn group with 1 update: [body-parser](https://github.com/expressjs/body-parser).


Updates `body-parser` from 1.20.2 to 1.20.3
- [Release notes](https://github.com/expressjs/body-parser/releases)
- [Changelog](https://github.com/expressjs/body-parser/blob/master/HISTORY.md)
- [Commits](https://github.com/expressjs/body-parser/compare/1.20.2...1.20.3)

---
updated-dependencies:
- dependency-name: body-parser
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump rollup in the npm_and_yarn group (#40886)

Bumps the npm_and_yarn group with 1 update: [rollup](https://github.com/rollup/rollup).


Updates `rollup` from 4.21.2 to 4.23.0
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.21.2...v4.23.0)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump eslint from 8.57.0 to 8.57.1 (#40843)

Bumps [eslint](https://github.com/eslint/eslint) from 8.57.0 to 8.57.1.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.57.0...v8.57.1)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump hugo-bin from 0.130.1 to 0.132.0 (#40882)

Bumps [hugo-bin](https://github.com/fenneclab/hugo-bin) from 0.130.1 to 0.132.0.
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.130.1...v0.132.0)

---
updated-dependencies:
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump npm-run-all2 from 6.2.2 to 6.2.3 (#40842)

Bumps [npm-run-all2](https://github.com/bcomnes/npm-run-all2) from 6.2.2 to 6.2.3.
- [Release notes](https://github.com/bcomnes/npm-run-all2/releases)
- [Changelog](https://github.com/bcomnes/npm-run-all2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bcomnes/npm-run-all2/compare/v6.2.2...v6.2.3)

---
updated-dependencies:
- dependency-name: npm-run-all2
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

XhmikosR committed:
Dependabot: switch to grouped updates (#40888)

dependabot[bot] committed:
Build(deps-dev): Bump terser from 5.31.6 to 5.34.1 (#40883)

Bumps [terser](https://github.com/terser/terser) from 5.31.6 to 5.34.1.
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/compare/v5.31.6...v5.34.1)

---
updated-dependencies:
- dependency-name: terser
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 9 updates (#40895)

* Build(deps-dev): Bump the development-dependencies group across 1 directory with 9 updates

Bumps the development-dependencies group with 9 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@docsearch/js](https://github.com/algolia/docsearch) | `3.6.1` | `3.6.2` |
| [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/HEAD/packages/commonjs) | `26.0.1` | `28.0.0` |
| [@rollup/plugin-node-resolve](https://github.com/rollup/plugins/tree/HEAD/packages/node-resolve) | `15.2.3` | `15.3.0` |
| [@rollup/plugin-replace](https://github.com/rollup/plugins/tree/HEAD/packages/replace) | `5.0.7` | `6.0.1` |
| [eslint-plugin-html](https://github.com/BenoitZugmeyer/eslint-plugin-html) | `8.1.1` | `8.1.2` |
| [nodemon](https://github.com/remy/nodemon) | `3.1.4` | `3.1.7` |
| [postcss](https://github.com/postcss/postcss) | `8.4.45` | `8.4.47` |
| [sass](https://github.com/sass/dart-sass) | `1.77.8` | `1.78.0` |
| [stylelint-config-twbs-bootstrap](https://github.com/twbs/stylelint-config-twbs-bootstrap) | `15.0.0` | `15.1.0` |



Updates `@docsearch/js` from 3.6.1 to 3.6.2
- [Release notes](https://github.com/algolia/docsearch/releases)
- [Changelog](https://github.com/algolia/docsearch/blob/main/CHANGELOG.md)
- [Commits](https://github.com/algolia/docsearch/compare/v3.6.1...v3.6.2)

Updates `@rollup/plugin-commonjs` from 26.0.1 to 28.0.0
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/commonjs/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/commonjs-v28.0.0/packages/commonjs)

Updates `@rollup/plugin-node-resolve` from 15.2.3 to 15.3.0
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/node-resolve/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/node-resolve-v15.3.0/packages/node-resolve)

Updates `@rollup/plugin-replace` from 5.0.7 to 6.0.1
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/replace/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/json-v6.0.1/packages/replace)

Updates `eslint-plugin-html` from 8.1.1 to 8.1.2
- [Changelog](https://github.com/BenoitZugmeyer/eslint-plugin-html/blob/main/CHANGELOG.md)
- [Commits](https://github.com/BenoitZugmeyer/eslint-plugin-html/compare/v8.1.1...v8.1.2)

Updates `nodemon` from 3.1.4 to 3.1.7
- [Release notes](https://github.com/remy/nodemon/releases)
- [Commits](https://github.com/remy/nodemon/compare/v3.1.4...v3.1.7)

Updates `postcss` from 8.4.45 to 8.4.47
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.45...8.4.47)

Updates `sass` from 1.77.8 to 1.78.0
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.77.8...1.78.0)

Updates `stylelint-config-twbs-bootstrap` from 15.0.0 to 15.1.0
- [Release notes](https://github.com/twbs/stylelint-config-twbs-bootstrap/releases)
- [Commits](https://github.com/twbs/stylelint-config-twbs-bootstrap/compare/v15.0.0...v15.1.0)

---
updated-dependencies:
- dependency-name: "@docsearch/js"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@rollup/plugin-commonjs"
  dependency-type: direct:development
  update-type: version-update:semver-major
  dependency-group: development-dependencies
- dependency-name: "@rollup/plugin-node-resolve"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: "@rollup/plugin-replace"
  dependency-type: direct:development
  update-type: version-update:semver-major
  dependency-group: development-dependencies
- dependency-name: eslint-plugin-html
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: nodemon
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: sass
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: stylelint-config-twbs-bootstrap
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>

* Fix new stylelint-config-recess-order issues (#40887)

* Pin sass to 1.78

If the issues are fixed, it should be updated again along with the caret operator.

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: XhmikosR <xhmikosr@gmail.com>

XhmikosR committed:
Update devDependencies

* jasmine       ^5.2.0  →   ^5.3.0
* stylelint    ^16.8.1  →  ^16.9.0

coliff committed:
Remove unneeded flexbug workaround (#40876)

This was only needed for Internet Explorer. Removing it saves a couple of bytes. :-)

REF: https://github.com/philipwalton/flexbugs#flexbug-4

XhmikosR committed:
package.json: ignore more deps

julien-deramond committed:
Docs: fix `$enable-css-grid` → `$enable-cssgrid` in 'Customize > Options' (#40818)

hannahiss committed:
Docs: Indicate links in foreign language in Translations page (#40767)

Co-authored-by: Julien Déramond <juderamond@gmail.com>
Co-authored-by: Patrick H. Lauke <redux@splintered.co.uk>

hannahiss committed:
Docs: Make "Edit in Stackblitz" links explicit in 'Examples' page (#40766)

Co-authored-by: Julien Déramond <juderamond@gmail.com>

MohamadSalman11 committed:
Docs: remove unused class in the homepage (#40900)

Co-authored-by: Julien Déramond <juderamond@gmail.com>

julien-deramond committed:
Docs: drop useless `.justify-content-start` class in homepage (#40914)

julien-deramond committed:
Examples: fix Cheatsheet examples horizontal scrolling (#40799)

julien-deramond committed:
Docs: precise how to add new `.order-*` classes in 'Layout > Columns' (#40821)

Co-authored-by: Tobi <TobiGr@users.noreply.github.com>
Co-authored-by: Louis-Maxime Piton <louismaxime.piton@orange.com>

louismaximepiton committed:
Docs: replace `.grid` by `.d-grid` in 'Utilities > Spacing' gap section (#40790)

Co-authored-by: Julien Déramond <juderamond@gmail.com>

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 6 updates (#40920)

Bumps the development-dependencies group with 6 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@babel/cli](https://github.com/babel/babel/tree/HEAD/packages/babel-cli) | `7.25.6` | `7.25.7` |
| [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) | `7.25.2` | `7.25.7` |
| [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) | `7.25.4` | `7.25.7` |
| [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) | `2.30.0` | `2.31.0` |
| [rollup](https://github.com/rollup/rollup) | `4.23.0` | `4.24.0` |
| [sass-true](https://github.com/oddbird/true) | `8.0.0` | `8.1.0` |



Updates `@babel/cli` from 7.25.6 to 7.25.7
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.7/packages/babel-cli)

Updates `@babel/core` from 7.25.2 to 7.25.7
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.7/packages/babel-core)

Updates `@babel/preset-env` from 7.25.4 to 7.25.7
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.7/packages/babel-preset-env)

Updates `eslint-plugin-import` from 2.30.0 to 2.31.0
- [Release notes](https://github.com/import-js/eslint-plugin-import/releases)
- [Changelog](https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md)
- [Commits](https://github.com/import-js/eslint-plugin-import/compare/v2.30.0...v2.31.0)

Updates `rollup` from 4.23.0 to 4.24.0
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.23.0...v4.24.0)

Updates `sass-true` from 8.0.0 to 8.1.0
- [Release notes](https://github.com/oddbird/true/releases)
- [Changelog](https://github.com/oddbird/true/blob/main/CHANGELOG.md)
- [Commits](https://github.com/oddbird/true/compare/v8.0.0...v8.1.0)

---
updated-dependencies:
- dependency-name: "@babel/cli"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: eslint-plugin-import
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: sass-true
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps): Bump the npm_and_yarn group with 2 updates (#40929)

Bumps the npm_and_yarn group with 2 updates: [cookie](https://github.com/jshttp/cookie) and [socket.io](https://github.com/socketio/socket.io).


Updates `cookie` from 0.4.2 to 0.7.2
- [Release notes](https://github.com/jshttp/cookie/releases)
- [Commits](https://github.com/jshttp/cookie/compare/v0.4.2...v0.7.2)

Updates `socket.io` from 4.7.5 to 4.8.0
- [Release notes](https://github.com/socketio/socket.io/releases)
- [Changelog](https://github.com/socketio/socket.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io/compare/socket.io@4.7.5...socket.io@4.8.0)

---
updated-dependencies:
- dependency-name: cookie
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: socket.io
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

MohamadSalman11 committed:
Docs: fix `background-blend-mode` CSS value (#40931)

louismaximepiton committed:
Add missing svgs

louismaximepiton reviewed (approved):
I'm fine having this in Bootstrap.

julien-deramond committed:
Docs: fix example hiding element except on medium and large devices in 'Utilities > Display > Hiding elements' (#40951)

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 6 updates (#40953)

Bumps the development-dependencies group with 6 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) | `7.25.7` | `7.25.8` |
| [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) | `7.25.7` | `7.25.8` |
| [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/HEAD/packages/commonjs) | `28.0.0` | `28.0.1` |
| [hugo-bin](https://github.com/fenneclab/hugo-bin) | `0.132.0` | `0.133.1` |
| [stylelint](https://github.com/stylelint/stylelint) | `16.9.0` | `16.10.0` |
| [terser](https://github.com/terser/terser) | `5.34.1` | `5.36.0` |



Updates `@babel/core` from 7.25.7 to 7.25.8
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.8/packages/babel-core)

Updates `@babel/preset-env` from 7.25.7 to 7.25.8
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.8/packages/babel-preset-env)

Updates `@rollup/plugin-commonjs` from 28.0.0 to 28.0.1
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/commonjs/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/commonjs-v28.0.1/packages/commonjs)

Updates `hugo-bin` from 0.132.0 to 0.133.1
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.132.0...v0.133.1)

Updates `stylelint` from 16.9.0 to 16.10.0
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/16.9.0...16.10.0)

Updates `terser` from 5.34.1 to 5.36.0
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/compare/v5.34.1...v5.36.0)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@rollup/plugin-commonjs"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: stylelint
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: terser
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

julien-deramond committed:
Docs: add ngx-bootstrap to JavaScript frameworks list (#40932)

MohamadSalman11 committed:
Docs: fix transparent navbar in Windows high contrast mode (#40911)

Co-authored-by: Julien Déramond <juderamond@gmail.com>

julien-deramond committed:
Docs: correct example formatting in 'Utilities > API' for linting compliance (#40973)

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 6 updates (#40978)

Bumps the development-dependencies group with 6 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@babel/cli](https://github.com/babel/babel/tree/HEAD/packages/babel-cli) | `7.25.7` | `7.25.9` |
| [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) | `7.25.8` | `7.25.9` |
| [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) | `7.25.8` | `7.25.9` |
| [hugo-bin](https://github.com/fenneclab/hugo-bin) | `0.133.1` | `0.133.5` |
| [npm-run-all2](https://github.com/bcomnes/npm-run-all2) | `6.2.3` | `7.0.1` |
| [vnu-jar](https://github.com/validator/validator) | `23.4.11` | `24.10.17` |



Updates `@babel/cli` from 7.25.7 to 7.25.9
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.9/packages/babel-cli)

Updates `@babel/core` from 7.25.8 to 7.25.9
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.9/packages/babel-core)

Updates `@babel/preset-env` from 7.25.8 to 7.25.9
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.25.9/packages/babel-preset-env)

Updates `hugo-bin` from 0.133.1 to 0.133.5
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.133.1...v0.133.5)

Updates `npm-run-all2` from 6.2.3 to 7.0.1
- [Release notes](https://github.com/bcomnes/npm-run-all2/releases)
- [Changelog](https://github.com/bcomnes/npm-run-all2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bcomnes/npm-run-all2/compare/v6.2.3...v7.0.1)

Updates `vnu-jar` from 23.4.11 to 24.10.17
- [Release notes](https://github.com/validator/validator/releases)
- [Changelog](https://github.com/validator/validator/blob/main/CHANGELOG.md)
- [Commits](https://github.com/validator/validator/commits)

---
updated-dependencies:
- dependency-name: "@babel/cli"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: npm-run-all2
  dependency-type: direct:development
  update-type: version-update:semver-major
  dependency-group: development-dependencies
- dependency-name: vnu-jar
  dependency-type: direct:development
  update-type: version-update:semver-major
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 5 updates (#41010)

Bumps the development-dependencies group with 5 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) | `7.25.9` | `7.26.0` |
| [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) | `7.25.9` | `7.26.0` |
| [@docsearch/js](https://github.com/algolia/docsearch) | `3.6.2` | `3.7.0` |
| [hugo-bin](https://github.com/fenneclab/hugo-bin) | `0.133.5` | `0.134.1` |
| [rollup](https://github.com/rollup/rollup) | `4.24.0` | `4.24.4` |



Updates `@babel/core` from 7.25.9 to 7.26.0
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.0/packages/babel-core)

Updates `@babel/preset-env` from 7.25.9 to 7.26.0
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.0/packages/babel-preset-env)

Updates `@docsearch/js` from 3.6.2 to 3.7.0
- [Release notes](https://github.com/algolia/docsearch/releases)
- [Changelog](https://github.com/algolia/docsearch/blob/main/CHANGELOG.md)
- [Commits](https://github.com/algolia/docsearch/compare/v3.6.2...v3.7.0)

Updates `hugo-bin` from 0.133.5 to 0.134.1
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.133.5...v0.134.1)

Updates `rollup` from 4.24.0 to 4.24.4
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.24.0...v4.24.4)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: "@docsearch/js"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

julien-deramond committed:
Docs: fix `.order` values in the migration guide (#41030)

patrickzzz committed:
Fix left alignment of floating labels with `.form-select` size variants (#41013)

Co-authored-by: Patrick Kroog <pk@kroog.com>
Co-authored-by: Christian Oliff <christianoliff@pm.me>
Co-authored-by: Julien Déramond <juderamond@gmail.com>
Co-authored-by: Louis-Maxime Piton <louismaxime.piton@orange.com>

hannahiss commented:
Hi @patrickhlauke could you please add your review? I added aria-hidden everywhere in the examples, like we talked about... 
Thanks 😄

vprothais committed:
Docs: Fix broken link in dropdowns documentation (#41052)

julien-deramond committed:
Docs: add warning callout for Yarn Berry for Bootstrap examples compat (#41036)

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 1</span>
+    <a href="#"><svg class="bi ms-1" width="16" height="16" role="img" aria-label="Delete primary 1"><use xlink:href="#x-circle-fill"/></svg></a>
```
for links that contain an svg, I would say it might be cleaner to add the `aria-label` to the link, rather than the svg. (the svg doesn't then need any `role` or `aria-hidden`, because the `aria-label` on the link will override/erase anything that's actually inside it)

patrickhlauke replied:
doing it on the link rather than the svg then means the link would be announced cleanly as "Delete primary 1, link" rather than "Delete primary 1, graphic, link" (as the fact that this is a "graphic" is really not that important/relevant to the user, I'd argue.

patrickhlauke replied:
```suggestion
    <a href="#" aria-label="Delete primary 1"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

dependabot[bot] committed:
Build(deps-dev): Bump the development-dependencies group across 1 directory with 7 updates (#41082)

Bumps the development-dependencies group with 7 updates in the / directory:

| Package | From | To |
| --- | --- | --- |
| [@babel/cli](https://github.com/babel/babel/tree/HEAD/packages/babel-cli) | `7.25.9` | `7.26.4` |
| [@docsearch/js](https://github.com/algolia/docsearch) | `3.7.0` | `3.8.0` |
| [hugo-bin](https://github.com/fenneclab/hugo-bin) | `0.134.1` | `0.136.3` |
| [postcss](https://github.com/postcss/postcss) | `8.4.47` | `8.4.49` |
| [rollup](https://github.com/rollup/rollup) | `4.24.4` | `4.28.1` |
| [stylelint](https://github.com/stylelint/stylelint) | `16.10.0` | `16.11.0` |
| [terser](https://github.com/terser/terser) | `5.36.0` | `5.37.0` |



Updates `@babel/cli` from 7.25.9 to 7.26.4
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.4/packages/babel-cli)

Updates `@docsearch/js` from 3.7.0 to 3.8.0
- [Release notes](https://github.com/algolia/docsearch/releases)
- [Changelog](https://github.com/algolia/docsearch/blob/main/CHANGELOG.md)
- [Commits](https://github.com/algolia/docsearch/compare/v3.7.0...v3.8.0)

Updates `hugo-bin` from 0.134.1 to 0.136.3
- [Release notes](https://github.com/fenneclab/hugo-bin/releases)
- [Commits](https://github.com/fenneclab/hugo-bin/compare/v0.134.1...v0.136.3)

Updates `postcss` from 8.4.47 to 8.4.49
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.4.47...8.4.49)

Updates `rollup` from 4.24.4 to 4.28.1
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v4.24.4...v4.28.1)

Updates `stylelint` from 16.10.0 to 16.11.0
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/16.10.0...16.11.0)

Updates `terser` from 5.36.0 to 5.37.0
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/compare/v5.36.0...v5.37.0)

---
updated-dependencies:
- dependency-name: "@babel/cli"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: "@docsearch/js"
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: hugo-bin
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: postcss
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development-dependencies
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: stylelint
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
- dependency-name: terser
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development-dependencies
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

XhmikosR committed:
Update devDependencies and regenerate package-lock.json

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 2</span>
+    <a href="#"><svg class="bi ms-1" width="16" height="16" role="img" aria-label="Delete primary 2"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary 2"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 3</span>
+    <a href="#"><svg class="bi ms-1" width="16" height="16" role="img" aria-label="Delete primary 3"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary 3"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Primary
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete primary"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Secondary
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete secondary"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete secondary"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Success
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete success"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete success"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Danger
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete danger"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete danger"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Warning
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete warning"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete warning"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Info
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete info"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete info"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Light
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete light"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete light"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Dark
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#"><svg class="bi" width="16" height="16" role="img" aria-label="Delete dark"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete dark"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke committed:
Move aria-label to wrapping links

avoids unnecessary "link, graphic, blah" announcement in screen readers

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -25,7 +25,7 @@
     <p class="col-md-4 mb-0 text-body-secondary">&copy; {{< year >}} Company, Inc</p>
 
     <a href="/" class="col-md-4 d-flex align-items-center justify-content-center mb-3 mb-md-0 me-md-auto link-body-emphasis text-decoration-none">
```
```suggestion
    <a href="/" class="col-md-4 d-flex align-items-center justify-content-center mb-3 mb-md-0 me-md-auto link-body-emphasis text-decoration-none" aria-label="Bootstrap">
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -25,7 +25,7 @@
 
     <a href="/" class="col-md-4 d-flex align-items-center justify-content-center mb-3 mb-md-0 me-md-auto link-body-emphasis text-decoration-none">
-      <svg class="bi me-2" width="40" height="32"><use xlink:href="#bootstrap"/></svg>
+      <svg class="bi me-2" width="40" height="32" role="img" aria-label="Bootstrap"><use xlink:href="#bootstrap"/></svg>
```
```suggestion
      <svg class="bi me-2" width="40" height="32" aria-hidden="true"><use xlink:href="#bootstrap"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -44,15 +44,15 @@
   <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
     <div class="col-md-4 d-flex align-items-center">
       <a href="/" class="mb-3 me-2 mb-md-0 text-body-secondary text-decoration-none lh-1">
```
```suggestion
      <a href="/" class="mb-3 me-2 mb-md-0 text-body-secondary text-decoration-none lh-1" aria-label="Bootstrap">
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -44,15 +44,15 @@
     <div class="col-md-4 d-flex align-items-center">
       <a href="/" class="mb-3 me-2 mb-md-0 text-body-secondary text-decoration-none lh-1">
-        <svg class="bi" width="30" height="24"><use xlink:href="#bootstrap"/></svg>
+        <svg class="bi" width="30" height="24" role="img" aria-label="Bootstrap"><use xlink:href="#bootstrap"/></svg>
```
```suggestion
        <svg class="bi" width="30" height="24" aria-hidden="true"><use xlink:href="#bootstrap"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -44,15 +44,15 @@
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#twitter"/></svg></a></li>
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
```
```suggestion
      <li class="ms-3"><a class="text-body-secondary" href="#" aria-label="Twitter"><svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#twitter"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -44,15 +44,15 @@
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Instagram"><use xlink:href="#instagram"/></svg></a></li>
```
```suggestion
      <li class="ms-3"><a class="text-body-secondary" href="#" aria-label="Instagram"><svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#instagram"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -44,15 +44,15 @@
-      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Instagram"><use xlink:href="#instagram"/></svg></a></li>
+      <li class="ms-3"><a class="text-body-secondary" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Facebook"><use xlink:href="#facebook"/></svg></a></li>
```
```suggestion
      <li class="ms-3"><a class="text-body-secondary" href="#" aria-label="Facebook"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -78,7 +78,7 @@
   <footer class="row row-cols-1 row-cols-sm-2 row-cols-md-5 py-5 my-5 border-top">
     <div class="col mb-3">
       <a href="/" class="d-flex align-items-center mb-3 link-body-emphasis text-decoration-none">
```
```suggestion
      <a href="/" class="d-flex align-items-center mb-3 link-body-emphasis text-decoration-none" aria-label="Bootstrap">
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -78,7 +78,7 @@
     <div class="col mb-3">
       <a href="/" class="d-flex align-items-center mb-3 link-body-emphasis text-decoration-none">
-        <svg class="bi me-2" width="40" height="32"><use xlink:href="#bootstrap"/></svg>
+        <svg class="bi me-2" width="40" height="32" role="img" aria-label="Bootstrap"><use xlink:href="#bootstrap"/></svg>
```
```suggestion
        <svg class="bi me-2" width="40" height="32" aria-hidden="true"><use xlink:href="#bootstrap"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -177,9 +177,9 @@ <h5>Subscribe to our newsletter</h5>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#twitter"/></svg></a></li>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
```
```suggestion
        <li class="ms-3"><a class="link-body-emphasis" href="#" aria-label="Twitter"><svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#twitter"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -177,9 +177,9 @@ <h5>Subscribe to our newsletter</h5>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Instagram"><use xlink:href="#instagram"/></svg></a></li>
```
```suggestion
        <li class="ms-3"><a class="link-body-emphasis" href="#" aria-label="Instagram"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/footers/index.html:
```diff
@@ -177,9 +177,9 @@ <h5>Subscribe to our newsletter</h5>
-        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Twitter"><use xlink:href="#twitter"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Instagram"><use xlink:href="#instagram"/></svg></a></li>
+        <li class="ms-3"><a class="link-body-emphasis" href="#"><svg class="bi" width="24" height="24" role="img" aria-label="Facebook"><use xlink:href="#facebook"/></svg></a></li>
```
```suggestion
        <li class="ms-3"><a class="link-body-emphasis" href="#" aria-label="Facebook"><svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#facebook"/></svg></a></li>
```

patrickhlauke commented on site/content/docs/5.3/examples/headers/index.html:
```diff
@@ -160,7 +160,7 @@ <h1 class="visually-hidden">Headers examples</h1>
     <div class="container-fluid d-grid gap-3 align-items-center" style="grid-template-columns: 1fr 2fr;">
       <div class="dropdown">
         <a href="#" class="d-flex align-items-center col-lg-4 mb-2 mb-lg-0 link-body-emphasis text-decoration-none dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">
```
```suggestion
        <a href="#" class="d-flex align-items-center col-lg-4 mb-2 mb-lg-0 link-body-emphasis text-decoration-none dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Bootstrap menu">
```

patrickhlauke commented on site/content/docs/5.3/examples/headers/index.html:
```diff
@@ -160,7 +160,7 @@ <h1 class="visually-hidden">Headers examples</h1>
       <div class="dropdown">
         <a href="#" class="d-flex align-items-center col-lg-4 mb-2 mb-lg-0 link-body-emphasis text-decoration-none dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">
-          <svg class="bi me-2" width="40" height="32"><use xlink:href="#bootstrap"/></svg>
+          <svg class="bi me-2" width="40" height="32" role="img" aria-label="Bootstrap menu"><use xlink:href="#bootstrap"/></svg>
```
```suggestion
          <svg class="bi me-2" width="40" height="32" aria-hidden="true"><use xlink:href="#bootstrap"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/product/index.html:
```diff
@@ -35,7 +35,7 @@ <h5 class="offcanvas-title" id="offcanvasLabel">Aperture</h5>
       <div class="offcanvas-body">
         <ul class="navbar-nav flex-grow-1 justify-content-between">
           <li class="nav-item"><a class="nav-link" href="#">
```
```suggestion
          <li class="nav-item"><a class="nav-link" href="#" aria-label="Aperture">
```

patrickhlauke commented on site/content/docs/5.3/examples/product/index.html:
```diff
@@ -35,7 +35,7 @@ <h5 class="offcanvas-title" id="offcanvasLabel">Aperture</h5>
         <ul class="navbar-nav flex-grow-1 justify-content-between">
           <li class="nav-item"><a class="nav-link" href="#">
-            <svg class="bi" width="24" height="24"><use xlink:href="#aperture"/></svg>
+            <svg class="bi" width="24" height="24" role="img" aria-label="Aperture"><use xlink:href="#aperture"/></svg>
```
```suggestion
            <svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#aperture"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/product/index.html:
```diff
@@ -44,7 +44,7 @@ <h5 class="offcanvas-title" id="offcanvasLabel">Aperture</h5>
           <li class="nav-item"><a class="nav-link" href="#">Support</a></li>
           <li class="nav-item"><a class="nav-link" href="#">Pricing</a></li>
           <li class="nav-item"><a class="nav-link" href="#">
```
```suggestion
          <li class="nav-item"><a class="nav-link" href="#" aria-label="Cart">
```

patrickhlauke commented on site/content/docs/5.3/examples/product/index.html:
```diff
@@ -44,7 +44,7 @@ <h5 class="offcanvas-title" id="offcanvasLabel">Aperture</h5>
           <li class="nav-item"><a class="nav-link" href="#">Pricing</a></li>
           <li class="nav-item"><a class="nav-link" href="#">
-            <svg class="bi" width="24" height="24"><use xlink:href="#cart"/></svg>
+            <svg class="bi" width="24" height="24" role="img" aria-label="Cart"><use xlink:href="#cart"/></svg>
```
```suggestion
            <svg class="bi" width="24" height="24" aria-hidden="true"><use xlink:href="#cart"/></svg>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 1</span>
+    <a href="#" aria-label="Delete primary 1"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary 1"><svg class="bi ms-1" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 2</span>
+    <a href="#" aria-label="Delete primary 2"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary 2"><svg class="bi ms-1" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -82,16 +82,16 @@
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 3</span>
+    <a href="#" aria-label="Delete primary 3"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary 3"><svg class="bi ms-1" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Primary
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete primary"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete primary"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Secondary
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete secondary"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete secondary"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Success
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete success"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete success"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Danger
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete danger"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete danger"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Warning
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete warning"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete warning"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Info
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete info"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete info"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Light
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete light"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete light"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke commented on site/content/docs/5.3/examples/badges/index.html:
```diff
@@ -102,48 +102,48 @@
     Dark
     <span class="vr mx-2"></span>
-    <a href="#"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <a href="#" aria-label="Delete dark"><svg class="bi" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```
```suggestion
    <a href="#" aria-label="Delete dark"><svg class="bi" width="16" height="16" aria-hidden="true"><use xlink:href="#x-circle-fill"/></svg></a>
```

patrickhlauke committed:
Merge branch 'main' into main-his-missing-aria-hidden

patrickhlauke commented:
did a big cleanup here https://github.com/twbs/bootstrap/pull/40756/commits/2e1f1161d1572ba700677b8f753ee84ee93f2f5d ... merging now (though I'd still say in most cases it's unnecessary to add `aria-hidden` when it's a) overriden by the `aria-label` of its parent/ancestor, or b) is completely empty of text nodes)
